package.json

```diff
@@ -234,11 +233,11 @@
     "@types/lru-cache": "^4.1.0",
     "@types/request-promise-native": "^1.0.14",
     "cheerio": "^1.0.0-rc.2",
-    "js-yaml": "^3.11.0",
-    "lodash": "^4.17.10",
+    "js-yaml": "^3.13.1",
+    "lodash": "^4.17.11",
     "lru-cache": "^4.1.3",
     "request": "^2.86.0",
     "request-promise-native": "^1.0.5",
-    "vscode-languageclient": "^4.4.0"
+    "vscode-languageclient": "5.3.0-next.4"
```

Comment:
This looks like an "under development" version?

Reply:
Yes, I'm not sure when it is planned on being released. This PR is not blocking anything though so we can wait until there is a stable release

Reply:
I would hate to break things for users out there.